### ISSUE TITLE
`plotting.plot_panel` fails with LazyFrames

### ISSUE THREAD

**Issue by baggiponte** [neocortexdb/functime#74]
This code results in the following error: `AttributeError: 'LazyFrame' object has no attribute 'get_column'`

```python
import polars as pl

y = pl.scan_parquet("https://github.com/descendant-ai/functime/raw/main/data/commodities.parquet")

plotting.plot_panel(y)
```

This is due to `get_column` being available for `pl.DataFrames` only. I think `plot_panel` could benefit from lazy operations since only number of entities * `last_n` points are plotted. The following line would just require adding a `.collect()` in the end if `y` is a `LazyFrame`.

https://github.com/neocortexdb/functime/blob/e02509c47882f556640b7c88299988cfbdde8836/functime/plotting.py#L54

**Comment by baggiponte** [comment]
Hey @topher-lo sorry for the ping, would you be okay with this feature? I can propose a PR in a breeze.

**Comment by topher-lo** [comment]
Makes sense to me. Just need to make sure the type hints are properly updated

**Comment by topher-lo** [comment]
Closed by #101
